src/components/ProjectSettingsTabs/GeneralTab/Fields/index.tsx

```diff
@@ -1,5 +1,4 @@
-import { FilesIgnoredFromCacheField } from "./FilesIgnoredFromCacheField";
 import { RepoConfigField } from "./RepoConfigField";
 import { RepotrackerField } from "./RepotrackerField";
 
-export { FilesIgnoredFromCacheField, RepoConfigField, RepotrackerField };
```

Comment:
Why do we want to remove this?

Reply:
I updated the field to use an approach managed entirely in the schema so no custom field is necessary! You can see the changes in `GeneralTab/getFormSchema.ts`